tsconfig.json

```diff
@@ -12,7 +13,9 @@
 		"rootDir": "src",
 		"sourceMap": true,
 		"strict": true,
-		"target": "es6"
+		"strictNullChecks": true,
+		"target": "ES6",
+		"types": ["node"]
```

Comment:
I think the `types` key should not be present. Otherwise, it negates all the `@types` packages we've saved as `devDependencies`. See: https://www.typescriptlang.org/docs/handbook/tsconfig-json.html#types-typeroots-and-types